develop/parachain-devs/interoperability/transfer-assets-with-xcm.md

```diff
@@ -0,0 +1,181 @@
+
+In the [Open Message Passing Channels](TODO:update-path){target=\_blank} tutorial, you saw how to open a two-way communication channel between chains by sending messages to the relay chain. You can use a similar strategy to send messages that allow a local chain to manage an account on a remote chain. In this tutorial, parachain B transfers assets into the sovereign account on the relay chain for parachain A.
+
+The outcome for this tutorial is similar to using the `transfer` function from the balances pallet, except in this case the transfer is initiated by a parachain and demonstrates how the holding register is used when executing the `WithdrawAsset` and `DepositAssetXCM` instructions.
+
+## Prerequisites
+
+Before you begin, verify the following:
+
+- You have the Sudo pallet available for both local parachains to use
+- You have opened the message passing channel to allow communication between parachain B and parachain A
+- You have set up a parachain test network using Zombienet or a local relay chain using the `rococo-local` chain specification. This tutorial uses the following Zombienet configuration file:
+
+    ```toml
+    [relaychain]
+    default_command = "polkadot"
+    chain = "rococo-local"
+
+    [[relaychain.nodes]]
+    name = "alice"
+    validator = true
+    ws_port = 9944
+
+    [[relaychain.nodes]]
+    name = "bob"
+    validator = true
+    ws_port = 9955
+
+    [[parachains]]
+    id = 1000
+
+    [parachains.collator]
+    name = "parachainA"
+    ws_port = 9988
+    command = "parachain-template-node"
+    args = [ "-l=xcm=trace" ]
+
+    [[parachains]]
+    id = 1001
+
+    [parachains.collator]
+    name = "parachainB"
+    ws_port = 9989
+    command = "parachain-template-node"
+    args = [ "-l=xcm=trace" ]
+
+    [settings]
+    timeout = 1000
+    ```
+
+    !!! note
+        Due to a well-known issue with Zombienet and HRMP channels, the opening of the HRMP channels between parachains has been forced by the relay chain through the `sudo` pallet for this tutorial. For further information about this workaround, please refer to [https://github.com/paritytech/polkadot-sdk/pull/1616#issuecomment-1727194584](https://github.com/paritytech/polkadot-sdk/pull/1616#issuecomment-1727194584){target=\_blank}
+
+## Configure XCM Instructions
+
+To illustrate the interaction between the two chains, in the following example, parachain B sends XCM instructions to deposit assets into an account on parachain A.
+
+1. Navigate to the **Extrinsics** tab in the Polkadot.Js Apps interface
+      1. Select the **Developer** tab
+      2. Choose the **Extrinsics** option from the dropdown menu
+
+        ![](/images/tutorials/polkadot-sdk/build-a-parachain/transfer-assets-with-xcm/transfer-assets-with-xcm-1.webp)
+
+2. Use the **sudo** pallet to send the XCM instructions
+      1. Select the **sudo** pallet from the dropdown menu
+      2. Choose the **sudo(call)** extrinsic to use the Sudo pallet to execute privileged transactions
```

Comment:
The name of the extrinsic is just `sudo`. I would not include `call` because, in other examples, we don't include the parameters
```suggestion
      2. Choose the **sudo** extrinsic to use the Sudo pallet to execute privileged transactions
```